planetwars/client/app/components/about/About.ts

```diff
@@ -1,18 +1,61 @@
 import * as React from 'react';
-import { button, div, h, p } from 'react-hyperscript-helpers';
+import {a, div, h, h1, h2, p} from 'react-hyperscript-helpers';
+import {Link} from "react-router-dom";
+
+// tslint:disable-next-line:no-var-requires
+const styles = require('./About.scss');
 
 interface IProps {
-  counter: number;
-  onIncrement: () => void;
 }
 
-interface IState { }
-
+interface IState {
+}
+// TODO: either make the link to github open externally or add a way to get back to the about page
 export default class About extends React.Component<IProps, IState> {
   public render() {
-    return div([
-      p([this.props.counter]),
-      button({ onClick: () => this.props.onIncrement() }, ['+1']),
+    return div(`.${styles.aboutPage}`, [
+      h1(`.${styles.aboutTitle}`, 'Welcome to PlanetWars!'),
+
+      h2(`.${styles.subTitle}`, 'Getting Started'),
+
+      p(['In order to start a game, you will need at least 2 bots. ' +
```

Comment:
Maybe mention this issue is temporal and easily fixed. We will probably ship some bots along, and people can create the same config twice.